index.js

```diff
@@ -21,6 +23,8 @@ if (Platform.OS === 'web') {
         VizbeePlayerState,
         VizbeeTrackStatus,
         VizbeeTrackInfo,
+        VizbeeSignInDelegate,
+        VizbeeSignInInfo,
```

Comment:
@naveenk1223 Can we move these 2 next to the VizbeeManager before VizbeeVideo just to be organized properly?

Reply:
@shivareddy Re-organized the exports as per flow